apps/web/src/pages/templates/components/email-editor/EmailCustomCodeEditor.tsx

```diff
@@ -68,9 +69,17 @@ export function EmailCustomCodeEditor({
         }}
       />
     </Card>
+    </div>
   );
 }
 
+const styledContainer = () => ({
+  height: '80vh', // Adjust the percentage as needed
```

Comment:
Setting a wrapper to be 80vh will not do the job, as the card itself -> the ace editor is set to a specific height (line 55). So what you did will  simply result in the editor height being as it is now, unchanged, but will add a lot of empty page + scroll